Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/WidgetService.java

```diff
@@ -123,6 +124,8 @@ private void showSilentNotification(Intent intent){
 
             if(entity_payload!=null && entity_payload.has("baseFare")) {
                 System.out.println("PAYLOAD + PAYLIAD " + entity_payload); // TODO:: REMOVE
+                mediaPlayer = MediaPlayer.create(getApplicationContext(), R.raw.silent_mode_notification_sound);
+                mediaPlayer.start();
```

Comment:
can you add a null check for mediaPlayer.start() with `mediaPlayer != null`

Reply:
Done